saleor/graphql/shop/types.py

```diff
@@ -222,16 +227,16 @@ def resolve_available_shipping_methods(_, info, channel, address=None):
         return resolve_available_shipping_methods(info, channel, address)
 
     @staticmethod
-    def resolve_countries(_, _info, language_code=None):
+    def resolve_countries(_, _info, language_code=None, in_shipping_zones=None):
```

Comment:
Shouldn't be `in_shupping_zones=False` by default?

Reply:
I added better description to this field. It can be None - in this case it returns all of the countries and that is what we want by default.